library/core/src/mem/mod.rs

```diff
@@ -642,24 +652,19 @@ pub unsafe fn zeroed<T>() -> T {
 /// is special in that the compiler knows that it does not have a fixed value.
 /// This makes it undefined behavior to have uninitialized data in a variable even
 /// if that variable has an integer type.
-/// (Notice that the rules around uninitialized integers are not finalized yet, but
-/// until they are, it is advisable to avoid them.)
```

Comment:
FWIW those rules are still not finalized, and I don't think there is lang team consensus here. (There hasn't really been further discussion since the comment was written.) As far as I know some lang team members are still taken aback by the idea that *integers* might care about the bit pattern (disallowing the "uninitialized" bitpattern).

Reply:
Sure, I can add this line back, I was just trying to make the language stronger and figured that this detail would be better left to a more authoritative document.

Reply:
I'm not sure if we have a more authoritative document.^^